include/spirv/unified1/spirv.core.grammar.json

```
@@ -5767,6 +5767,38 @@
       "extensions" : [ "SPV_NV_mesh_shader" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpFetchMicroTriangleVertexPositionNV",
+      "class"  : "Reserved",
+      "opcode" : 5300,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Accel'" },
+        { "kind" : "IdRef", "name" : "'Instance Id'" },
+        { "kind" : "IdRef", "name" : "'Geometry Id'" },
+        { "kind" : "IdRef", "name" : "'Primitive Id'" },
```

In the spec these are named differently:
```suggestion
        { "kind" : "IdRef", "name" : "'Geometry Index'" },
        { "kind" : "IdRef", "name" : "'Primitive Index'" },
```